recipes/snippy/3.2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "snippy" %}
+{% set version = "3.2" %}
+{% set hash = "75377acc4687e21d46d773936389e3f0abd8441076d3c9acdd31be13b53352a8" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  sha256: {{ hash }}
+  url: https://github.com/tseemann/{{ name }}/archive/v{{ version }}.tar.gz
+
+build:
+  number: 1
```

Comment:
Please bump this to 2.

Reply:
Do i have to manually increment this at every build *attempt* ?

Reply:
Hum, you only need to increment it once in the same pull request. It's only when the pull request is merged that the package with this build number is uploaded to anaconda.org